daemon/src/auto_rollover.rs

```diff
@@ -7,7 +7,6 @@ use crate::model::cfd::OrderId;
 use crate::oracle;
```

Comment:
@da-kami Most of the changes to this file are because I used a version that @klochowicz and I worked on which is almost exactly the same as what you ended up doing, so not much to see here.